SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -92,15 +92,27 @@ Each of the `spdlog` source files containing code include a copyright statement
 
 ### Feature Testing [4.i]
 
-Many (if not all) of the advertised features of `spdlog` are tested as part of continuous integration.
+Each feature in spdlog has corresponding tests which simulate typical usage, and they are located in link to test source code. The result of these tests can be found at:
```

Comment:
@ahcorde , sorry, I meant that we should link to where the tests live where it says "link to test source code"